controller/src/main/java/io/pravega/controller/task/Stream/StreamMetadataTasks.java

```diff
@@ -610,7 +611,8 @@ public void setRequestEventWriter(EventStreamWriter<ControllerEvent> requestEven
                 });
     }
 
-    private CompletableFuture<CreateStreamStatus.Status> createStreamBody(String scope, String stream,
+    @VisibleForTesting
+    CompletableFuture<CreateStreamStatus.Status> createStreamBody(String scope, String stream,
```

Comment:
Is there a different way to create a createStream request for tests that does not require changing the visibility of this method? This is probably the most obvious way, but I'm wondering if we can test the same functionality making a higher-level call, e.g., by issuing a real request to create a stream to the tested controller part.

Reply:
Ok.